portfolio/src/main/webapp/index.html

```diff
@@ -12,14 +12,14 @@ <h1>Jacinda's Portfolio</h1>
       <p>Welcome to my portfolio.</p>
       <img src = "/images/tenor.gif" />
 
-    <!--Button currently not connecting with url-->
-      <button onclick = "ShowSomething();">
+    <!--Button currently not working-->
+      <button onclick = "showSomething();">
         Click here for a surprise! :)
     </button>
     <p id="servlet-container"></p>
        
-     <!---->
-    <p>Click <a href="/hello">here</a> to see the website too.</p>
+     <!--<p>Click <a href="/hello">here</a> to see the website too.</p>-->
```

Comment:
Now that you're using git and committing your code regularly, you could actually just delete this line entirely instead of commenting it out - you'll be able to find the previously-committed version later if you want it, with full context explaining its edit history etc. ;)